index.html

```diff
@@ -2,9 +2,20 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title></title>
+    <link rel="stylesheet" href="./app.css">
+    <title>Popular Github</title>
   </head>
   <body>
-
+      <div class="profile-photo">
+        <div class="profile-name">
```

Comment:
ok, cool... Yea, still wrapping my brain around the whole grouping elements... when to do it, why you do it... also, you mentioned sub element... is a sub element a block within a block??